sdk/netapp/arm-netapp/src/models/index.ts

```diff
@@ -699,6 +676,73 @@ export interface VolumePatch extends BaseResource {
   exportPolicy?: VolumePatchPropertiesExportPolicy;
 }
 
+/**
+ * Mount Target
+ */
+export interface MountTarget {
```

Comment:
Where is this interface being used? I see 0 references.

Reply:
yes, no references now, and still exist in the swagger definition